### PR TITLE
Replaced getMarketParticipantActorActorCredentials with graphql

### PR DESCRIPTION
**Pull request by mimse**
https://app.zenhub.com/workspaces/raccoons-sprint-board-60d029726fc7570010362aa7/issues/zh/476

### CODE REVIEW

File: libs/dh/shared/data-access-mocks/src/lib/market-participant.ts [hunk 1]
```diff
@@ -390,21 +386,6 @@ function getAuditLogByActorId() {
...
-      expirationDate: '2020-09-30T12:00:00',
-    },
-  };
-
-  return http.get(`${apiBase}/v1/MarketParticipantActor/GetActorCredentials`, async () => {
-    await delay(mswConfig.delay);
-    return HttpResponse.json(response);
-  });
-}
-
```
Comment (dzhavat) [inline libs/dh/shared/data-access-mocks/src/lib/market-participant.ts:407]: suggestion: add a mock for `GetActorCredentials` query

Comment (mimse) [reply libs/dh/shared/data-access-mocks/src/lib/market-participant.ts:407]: Done in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3376/commits/31088b3e2b2ef745bc167a00b82b71b36349fc59

File: libs/dh/market-participant/actors/data-access-api/src/lib/dh-market-party-b2b-access.store.ts [hunk 2]
```diff
@@ -19,15 +19,20 @@ import { ComponentStore } from '@ngrx/component-store';
...
 import { ApiErrorCollection } from '@energinet-datahub/dh/market-participant/data-access-api';
+import { GetActorCredentialsDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+
+import type { ResultOf } from '@graphql-typed-document-node/core';
+
+type ActorCredentials = ResultOf<typeof GetActorCredentialsDocument>['actorById'];
 
 interface DhB2BAccessState {
-  credentials: MarketParticipantActorCredentialsDto | null;
+  credentials: ActorCredentials | null | undefined;
```
Comment (dzhavat) [inline libs/dh/market-participant/actors/data-access-api/src/lib/dh-market-party-b2b-access.store.ts:35]: suggestion: if we use `ActorCredentialsDto` type here, there will be no need to use `credentials?.credentials` (double `credentials`) later on.

Comment (mimse) [reply libs/dh/market-participant/actors/data-access-api/src/lib/dh-market-party-b2b-access.store.ts:35]: Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3376/commits/54b8cfe8147febd03616088ca42d951a8486c176

**Review by dzhavat — APPROVED** [review]

**Pull request merged**